### ISSUE TITLE
saving

### ISSUE THREAD

**Issue by wouterk1MSS** [koaning/bulk#3]
great stuff, very useful thanks!

I noticed for me saving doesn't work yet when using the packaged version- it just creates an empty csv file. tested this on 2 systems. It does work when running bokeh serve scripts/bulk_text.py --show

(ps1. note that in the bulk_text.py file it still refers to "meant to be ran via: bokeh serve scripts/main.py --show )

(ps2. there is still a reference in readme to "original.csv" which i believe now refers to cluestarred.csv)

**Comment by koaning** [comment]
Ah, that's a total bug on my end, I think.  

Just to check, can you see anything of a traceback?

**Comment by wouterk1MSS** [comment]
nope none that I could see

**Comment by koaning** [comment]
Ah yeah. I'm able to confirm the error locally. 

I also noticed that the installation step does not install `pandas`, which I gotta fix. And I also noticed that we don't print the ip/port number where the bulk server is running.

**Comment by koaning** [comment]
I just found a fix and pushed it. Could you let me know if it now works?

**Comment by wouterk1MSS** [comment]
I updated to v0.02, but still have the same issue, the content of out.csv is limited to ",text"

**Comment by koaning** [comment]
That is strange. Because I just installed it in a clean env and got it to work. Can you call `python -m pip freeze | grep bulk`? Also, what operating system are you on?

**Comment by koaning** [comment]
@JulesBelveze just to check, did you encounter this behavior?

**Comment by JulesBelveze** [comment]
@koaning Hmmm I just checked and it works as expected with `bulk==0.0.2`

**Comment by wouterk1MSS** [comment]
python -m pip freeze | grep bulk
bulk==0.0.2
This is on MacOS 12. 

I can confirm it works as expected on my windows installation

**Comment by koaning** [comment]
I'm running on a Macbook M1 Air with MacOS 12 and I'm not seeing this error. 

How are you running bulk? Via `python -m bulk`?

**Comment by wouterk1MSS** [comment]
yes I was running python -m bulk text ready.csv 

But I was inside the dir called bulk. I had also cloned the repo before just installing the package.
So it seems because it is running the old code still? Sorry perhaps this is obvious but not directly to me.

Anyway if i create a new separate called bbulk containing that csv, it does indeed give the desired behaviour. So the issue has been resolved.
